eth/chains/base.py

```diff
@@ -113,6 +117,14 @@ class BaseChain(Configurable, ChainAPI):
     chaindb_class: Type[ChainDatabaseAPI] = None
     vm_configuration: Tuple[Tuple[BlockNumber, Type[VirtualMachineAPI]], ...] = None
     chain_id: int = None
+    consensus_engine: VirtualMachineModifierAPI = None
```

Comment:
Looks like these should be added to the `ChainAPI`

Reply:
Good point! Fixed